templates/group.j2

```diff
@@ -1,8 +1,6 @@
 {% extends 'base.j2' %}
 
 {% block main %}
-  <h2 data-sassdoc="page-title">{{ pageTitle }}</h2>
```

Comment:
Ahh, both of these are because I didn't finish the breadcrumb Sondra requested in place of that mid-level header. PR now at #104